packages/node/test/machine/integration/protocols/test-runner.ts

```diff
@@ -0,0 +1,414 @@
+import IdentityApp from "@counterfactual/cf-funding-protocol-contracts/build/IdentityApp.json";
+import { OutcomeType } from "@counterfactual/types";
+import { Contract, ContractFactory } from "ethers";
+import { One, Two, Zero } from "ethers/constants";
+import { JsonRpcProvider } from "ethers/providers";
+import { BigNumber, getAddress } from "ethers/utils";
+
+import { CONVENTION_FOR_ETH_TOKEN_ADDRESS } from "../../../../src/constants";
+import { Protocol, xkeyKthAddress } from "../../../../src/machine";
+import { sortAddresses } from "../../../../src/machine/xkeys";
+import { getBalancesFromFreeBalanceAppInstance } from "../../../../src/models/free-balance";
+import { getCreate2MultisigAddress } from "../../../../src/utils";
+import { toBeEq } from "../bignumber-jest-matcher";
+import { connectToGanache } from "../connect-ganache";
+import { MessageRouter } from "../message-router";
+import { MiniNode } from "../mininode";
+
+expect.extend({ toBeEq });
+
+export enum Participant {
+  A,
+  B,
+  C
+}
+
+export class TestRunner {
+  static readonly TEST_TOKEN_ADDRESS = getAddress(
+    "88a5c2d9919e46f883eb62f7b8dd9d0cc45bc290"
+  );
```

Comment:
You should not need to run `getAddress` on a string literal.

Reply:
`getAddress(
    "88a5c2d9919e46f883eb62f7b8dd9d0cc45bc290"
  )`

is equal to 

`"0x88a5c2d9919e46f883eb62f7b8dd9d0cc45bc290"`

seems like unnecessary complexity